### PR TITLE
Chore/image optimization

### PR DESCRIPTION
**Pull request by Keitio**

### CODE REVIEW

File: src/components/decorations/MamaStar.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import mamaImg from '../../../public/ui/actor/ma001001_01_actor_icon.png'
+import Image from 'next/image'
+
+export default function MamaStar({...all}): JSX.Element {
```
Comment (KeziahMoselle) [inline src/components/decorations/MamaStar.tsx:4]: detail: the convention is `...props`

File: src/components/decorations/Rank.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import rankA from '../../../public/ui/search/search_rank_2.png'
+import rankB from '../../../public/ui/search/search_rank_3.png'
+import rankC from '../../../public/ui/search/search_rank_4.png'
+import rankD from '../../../public/ui/search/search_rank_5.png'
+import rankE from '../../../public/ui/search/search_rank_6.png'
+import Image from 'next/image'
+
+type ranks = "S" | "A" | "B" | "C" | "D" | "E";
+
+export default function Rank({rank, ...all}: {rank: ranks}): JSX.Element {
```
Comment (KeziahMoselle) [inline src/components/decorations/Rank.tsx:11]: detail: the convention is `...props`

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ const { description, cover, url } = meta;
 class MyDocument extends Document {
   render() {
     return (
-      <Html>
+      <Html lang="en">
```
Comment (KeziahMoselle) [inline src/pages/_document.tsx:9]: 👍

File: src/styles/index.scss [hunk 4]
```diff
@@ -129,7 +124,7 @@ header > div {
   background-color: theme("colors.beige.darker");
   width: 56px;
   height: 56px;
-  margin-left: 24px;
+  // margin-left: 24px;
```
Comment (KeziahMoselle) [inline src/styles/index.scss:127]: detail: remove the comment if it's unecessary

**Review by KeziahMoselle — APPROVED** [review]

File: src/styles/index.scss (hunk 4 above)
Comment (Keitio) [reply src/styles/index.scss:127]: oh yeah missed that thanks

**Pull request merged**